netbox/ipam/forms/model_forms.py

```diff
@@ -818,13 +814,28 @@ def __init__(self, *args, **kwargs):
         if (parent_object_type_id := get_field_value(self, 'parent_object_type')):
```

Comment:
Opportunistic cleanup
```suggestion
        if parent_object_type_id := get_field_value(self, 'parent_object_type'):
```